tensorflow_serving/model_servers/server.cc

```diff
@@ -182,8 +182,8 @@ void Server::PollFilesystemAndReloadConfig(const string& config_file_path) {
 }
 
 Status Server::BuildAndStart(const Options& server_options) {
-  if (server_options.grpc_port == 0) {
-    return errors::InvalidArgument("server_options.grpc_port is not set.");
+  if (server_options.grpc_port <= 0 && server_options.grpc_socket_path.empty()) {
```

Comment:
use `== 0` instead of `<= 0`

i'd retain the older behavior of treating `== 0` as grpc_port not being specified (that leaves room for adding yet another interpretation for `< 0` if we ever need to (i hope not ;))